test.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 
 echo "running linters and tests..."
-flake8 . && python -m unittest discover tests
+flake8 --ignore=E722,E501 . && python -m unittest discover tests
```

Comment:
Might be preferable to move these into a top level `.flake8` file like we have in pshtt: https://github.com/dhs-ncats/pshtt/blob/master/.flake8

That way if `flake8` is called manually or via some other method, you get the same ignores / results.

Reply:
Actually, we already had one! I just forgot about it entirely. Fixed in https://github.com/18F/domain-scan/commit/994cb3892f42be4f91b3555152d8cf2b41974cc2, thanks for catching that.